_includes/header.html

```diff
@@ -1,9 +1,15 @@
 <header>
   <div class="pure-g">
-    <div class="pure-u-1 pure-u-lg-1-5">
-        <a href="{{ site.baseurl }}/">
-          <img class="nav-icon" src={{ site.logo }} alt="MyPaint" />
-        </a>
+    <div class="logo pure-u-1 pure-u-lg-1-5">
+      <a href="{{ site.url }}">
```

Comment:
Oh. I though that was normal since it allowed me to use my localhost ip as the base when I launch `jekyll serve`. Plus it was what written on the [original site](https://github.com/mypaint/mypaint.github.io/blob/master/_includes/header.html#L5) so that was what I based it off.